context/app/static/js/components/Search/Search.scss

```diff
@@ -7,6 +7,15 @@ $secondary-gray: #636363;
     margin: 16px 0px 32px 0px;
 }
 
+.sk-selected-filters {
+    // Together with 8px margin on SelectedFilterDiv, gives 16px between elements
+    margin-top: 8px;
+}
+
+.sk-selected-filters-option__remove-action {
+    border: none;
+}
+
```

Comment:
Are these styles for the element that contains the selected filters which is only accessible through extending the searchkit styles?

Reply:
Good catch: the `remove-action` isn't being used at all now, but `.sk-selected-filters` I think is still the best of doing that: We've overriden `SelectedFilter`, singular, but are still using the SK parent `SelectedFilters` plural.